pom.xml

```diff
@@ -104,7 +104,7 @@
     <license.skip>true</license.skip>
     <docker.repository>apache/drill</docker.repository>
     <antlr.version>4.8-1</antlr.version>
-    <maven.version>3.6.3</maven.version>
+    <maven.version>3.8.4</maven.version>
```

Comment:
Could you try to build the project with this version? Is that used for current CI builds?

Reply:
@vdiravka we have set GitHub ci to use `ubuntu-latest` which at the moments means Ubuntu 20.04 which has a Maven package version of 3.6.3-1.

Reply:
@vdiravka, it builds fine for me in Maven 3.8.4 too.

Reply:
Could you update the correspondent maven.md doc?

Reply:
You can update `Envirnment.md` with links to recommended and minimum Maven versions :
```
...
To build, you need to have the following software installed on your system to successfully complete a build. 
  * Java 8
  * Maven 3.6.3 or greater
```